README.md

```diff
@@ -46,15 +46,15 @@ Run indiviudal test
 
 Run SHC examples
 
-    ./bin/spark-submit --verbose --class org.apache.spark.sql.execution.datasources.hbase.examples.HBaseSource --master yarn-cluster --packages com.hortonworks:shc-core:1.1.1-2.1-s_2.11 --repositories http://repo.hortonworks.com/content/groups/public/ --files /usr/hdp/current/hbase-client/conf/hbase-site.xml shc-examples-1.1.1-2.1-s_2.11-SNAPSHOT.jar
+    ./bin/spark-submit --verbose --class org.apache.spark.sql.execution.datasources.hbase.examples.HBaseSource --master yarn-cluster --packages org.virtuslab:shc-core:1.1.1-2.1-s_2.11 --repositories https://repo1.maven.org/maven2/ --files /usr/hdp/current/hbase-client/conf/hbase-site.xml shc-examples-1.1.1-2.1-s_2.11-SNAPSHOT.jar
 
 The following illustrates how to run your application in real hbase cluster. You need to provide the hbase-site.xml. It may subject to change based on your specific cluster configuration.
 
-    ./bin/spark-submit  --class your.application.class --master yarn-client  --packages com.hortonworks:shc-core:1.1.1-2.1-s_2.11 --repositories http://repo.hortonworks.com/content/groups/public/ --files /etc/hbase/conf/hbase-site.xml /To/your/application/jar
+    ./bin/spark-submit  --class your.application.class --master yarn-client  --packages org.virtuslab:shc-core:1.1.1-2.1-s_2.11 --repositories https://repo1.maven.org/maven2/ --files /etc/hbase/conf/hbase-site.xml /To/your/application/jar
```

Comment:
Shouldn't we change the version if we added a feature?

Reply:
Not right now. We need to release it